specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/ServiceEndpointPolicyGet.json

```diff
@@ -0,0 +1,36 @@
+{
+  "parameters" : {
+    "api-version": "2018-06-01",
+    "subscriptionId" : "subid",
+    "resourceGroupName": "rg1",
+    "serviceEndpointPolicyName": "testServiceEndpointPolicy"
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "name" : "testServiceEndpointPolicy",
+        "id" : "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/serviceEndpointPolicies/testServiceEndpointPolicy",
+        "location" : "westus",
+        "properties": 
+         {
+          "serviceEndpointPolicyDefinitions": [
+          {
+            "name": "StorageServiceEndpointPolicyDefinition",
+            "properties": {
+             "description": "Storage Service EndpointPolicy Definition",
+             "service": "Microsoft.Storage",
+             "serviceResources": [
+                "/subscriptions/subid1",
+                "/subscriptions/subid1/resourceGroups/storageRg",
+                "/subscriptions/subid1/resourceGroups/storageRg/providers/Microsoft.Storage/storageAccounts/stAccount"
+             ]
+            },
+           "provisioningState" : "Succeeded"
```

Comment:
`provisioningState` is not defined in the `ServiceEndpointPolicyDefinition` definition